src/profile_manager/forms.py

```diff
@@ -28,8 +34,60 @@ def clean_alias(self):
 
     def save(self, *args, **kwargs):
         super(ProfileForm, self).save(*args, **kwargs)
+
         user = self.instance.user
         user.email = self.cleaned_data['email']
         user.save()
 
         return self.instance
+
+
+class ProfileFormStaffOnly(ProfileForm):
+
+    username = forms.CharField(required=True)
+    first_name = forms.CharField(required=False)
+    last_name = forms.CharField(required=False)
+    staff = forms.BooleanField(required=False)
+    superuser = forms.BooleanField(required=False)
+    active = forms.BooleanField(required=False)
+
+    class Meta(ProfileForm.Meta):
+        fields = ProfileForm._meta.fields + [
+            'username', 'first_name', 'last_name', 'staff', 'superuser', 'active', 'is_TA',
+        ]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.fields['username'].initial = self.instance.user.username
+        self.fields['username'].help_text = "Unique usernames only!"
```

Comment:
This is what the default help text is, preferably you can just get this from the User model (User.
![image](https://user-images.githubusercontent.com/10604391/177474051-801566bc-b6e7-400e-b755-bde424e6beca.png)

I think this is a bit too sloppy for a key field such as username.  See if you can tap into the actual field used by username and use that instead of a charfield with none of that default validation.

For example, this could break if someone tries to enter more than 150 chars, because the default CHarfield might have a different limit (the form might accept >150, but the database will reject it)

If you can just get the username's default formfield, it should all be handled:  Maybe this?
https://stackoverflow.com/questions/2584141/how-to-get-django-form-field-from-model-field